service-providers/network-requester/src/config/template.rs

```diff
@@ -102,7 +102,6 @@ upstream_exit_policy_url = '{{ network_requester.upstream_exit_policy_url }}'
 [debug]
 
 [debug.traffic]
-average_packet_delay = '{{ debug.traffic.average_packet_delay }}'
 message_sending_average_delay = '{{ debug.traffic.message_sending_average_delay }}'
```

Comment:
I'd say also remove this one (so the whole `[debug.traffic]` section), because I've got a feeling in few weeks/months we're going to run into exactly the same problem, but for the poisson rates : D

Reply:
Done